src/output.cpp

```diff
@@ -39,8 +40,92 @@ header(const char* msg, int level) {
   }
 }
 
+std::string time_stamp()
+{
+  int base_year = 1990;
+  std::stringstream ts;
+  std::time_t t = std::time(0);   // get time now
+  std::tm* now = std::localtime(&t);
+  ts << now->tm_year + base_year << "-" << now->tm_mon
+     << "-" << now->tm_mday << " " << now->tm_hour
+     << ":" << now->tm_min << ":" << now->tm_sec;
+  return ts.str();
+}
+  
 //==============================================================================
 
+//===============================================================================
+// PRINT_PLOT displays selected options for plotting
+//===============================================================================
+
+void print_plot() {
+
+  header("PLOTTING SUMMARY", 5);
+
+  for (auto pl : plots) {
+    // Plot id
+    std::cout << "Plot ID: " << pl->id << std::endl;
+    // Plot filename
+    std::cout << "Plot file: " << pl->path_plot << std::endl;
+    // Plot level
+    std::cout << "Universe depth: " << pl->level << std::endl;
+
+    // Plot type
+    if (PLOT_TYPE::SLICE == pl->type) {
```

Comment:
This is a new style thing to me. In the past I've coded "equals" conditionals this way for safety (in case one of the `=` is missed). Are we rather against this?